lib/assessment.js

```diff
@@ -44,27 +45,30 @@ module.exports = {
     },
 
     /**
-     * Render the "text" property of an assessment.
+     * Render the `html` or `text` property of an assessment.
      *
-     * @param {Object} assessment - The assessment to render the text for.
+     * @param {import('../sync/course-db').Assessment} assessment - The assessment to render the text for.
      * @param {string} urlPrefix - The current server urlPrefix.
      * @param {function} callback - A callback(err, html) function.
      */
-    renderText(assessment, urlPrefix, callback) {
-        if (!assessment.text) return callback(null, null);
+    renderHtmlOrText(assessment, urlPrefix) {
```

Comment:
There was no reason for this function to be async, so I dropped the callback in favor of just returning a value.